it/src/test/java/org/eclipse/jkube/integrationtests/vertx/VertxK8sITCase.java

```diff
@@ -37,13 +43,7 @@
 import static org.eclipse.jkube.integrationtests.assertions.DockerAssertion.assertImageWasRecentlyBuilt;
 import static org.eclipse.jkube.integrationtests.assertions.YamlAssertion.yaml;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.allOf;
-import static org.hamcrest.Matchers.anEmptyMap;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.hasItems;
-import static org.hamcrest.Matchers.hasProperty;
-import static org.hamcrest.Matchers.hasSize;
-import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.*;
```

Comment:
We're using Eclipse import style. If you're in IntelliJ Change your settings:
Class count to use import with '*': 999
Names count to use static import with '*': 999
![image](https://user-images.githubusercontent.com/488391/78368188-847bcd80-75c3-11ea-9fe8-5a624d854ba5.png)